src/test/scala/io/iohk/ethereum/consensus/pow/EthashUtilsSpec.scala

```diff
@@ -18,8 +20,21 @@ class EthashUtilsSpec extends AnyFlatSpec with Matchers with ScalaCheckPropertyC
   val ecip1099forkBlockNumber: Long = 11460000
 
   "Ethash" should "generate correct hash" in {
-    forAll(Gen.choose[Long](0, 15000000L)) { blockNumber =>
-      seed(blockNumber, ecip1099forkBlockNumber) shouldBe seedForBlockReference(blockNumber)
+    val seedEpoch0 = ByteStringUtils.string2hash("0000000000000000000000000000000000000000000000000000000000000000")
+    val seedEpoch1 = ByteStringUtils.string2hash("290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563")
+    val seedEpoch382 = ByteStringUtils.string2hash("d3d0aa11197dcdcfcb3ad3c73d415af47299bddb47fda6081d31d9dd06462f6a")
+    val seedEpoch383 = ByteStringUtils.string2hash("bf532874eb434842e7a3e4acd113fe454541651872760d9b95d11d7f90ca25dc")
```

Comment:
where did these hashes come from? maybe this would be a good inspiration https://github.com/iquidus/ecip-1099-data ?

Reply:
These are the seeds that correspond to the blocks, calculated with the Mantis code found to be working properly (valid blocks are valid.)

But the strings themselves are not so interesting, just that they are the same across different blocks within a (ECIP-1099) epoch.

Reply:
(Also, the hash being all zeroes for epoch 0 is a good sign?)